apps/api/src/app/subscribers/dto/create-subscriber.dto.ts

```diff
@@ -26,4 +26,9 @@ export class CreateSubscriberBodyDto {
   @IsString()
   @IsOptional()
   avatar?: string;
+
+  @IsArray()
+  @IsString({ each: true })
+  @IsOptional()
+  notificationIdentifiers?: string[];
```

Comment:
This one we will have to integrate with @djabarovgeorge's code on Direct channel.